ext/reflection/tests/closures_005.phpt

```diff
@@ -0,0 +1,28 @@
+--TEST--
+Test closure isStatic
+--FILE--
+<?php
+class Foo {
+    public static function bar(){}
+    public function baz() {}
+    
+    public function qux() {
+        return static function(){};
+    }
+}
+
+$foo = new Foo;
+
+var_dump(
+    (new ReflectionFunction(function(){}))->isStatic(),
+    (new ReflectionFunction(static function(){}))->isStatic(),
```

Comment:
fn should be probably tested as well

```suggestion
    (new ReflectionFunction(fn() => false))->isStatic(),
    (new ReflectionFunction(static fn() => false))->isStatic(),
```

Reply:
Wouldn't provide any additional coverage.